### ISSUE TITLE
[de] incorrect tagging "Überraschungsmoment"

### ISSUE THREAD

**Issue by tiff** [languagetool-org/languagetool#3315]
> Der Überraschungsmoment

![Bildschirmfoto 2020-07-21 um 18 01 51](https://user-images.githubusercontent.com/37363/88078105-54f5ce80-cb7c-11ea-8ef4-55eecf3cc389.png)

@danielnaber wieder mal was für dich. Bestimmt Splitting kaputt, oder?

**Comment by danielnaber** [comment]
Nein, LT hat schon recht: https://www.duden.de/suchen/dudenonline/%C3%9Cberraschungsmoment und https://www.korrekturen.de/genus/der-oder-das-ueberraschungsmoment.shtml - Regel wäre aber gut, die das erklärt

**Comment by janschreiber** [comment]
> Regel wäre aber gut, die das erklärt

Interessantes Thema. Hier schon mal eine Liste mit Wortkandidaten für eine Regel:

Abschaltdrehmoment
Anfangsdrehmoment
Anlaufdrehmoment
Anlaufmoment
Antriebsdrehmoment
Antriebsmoment
Anziehdrehmoment
Anziehmoment
Anzugsdrehmoment
Anzugsmoment
Ausgangsdrehmoment
Auslösemoment
Bemessungsdrehmoment
Beschleunigungsmoment
Biegemoment
Dauerdrehmoment
Deviationsmoment
Differenzmoment
Dipolmoment
Direktionsmoment
Drehmoment
Druckmoment
Eigendrehmoment
Eingangsdrehmoment
Einspannmoment
Enddrehmoment
Flächenträgheitsmoment
Flächenzentrifugalmoment
Gesamtdrehmoment
Gewichtsmoment
Hebelmoment
Hochlaufmoment
Impulsmoment
Inertialmoment
Kippmoment
Kraftmoment
Kupplungsdrehmoment
Kupplungsmoment
Lastmoment
Lenkdrehmoment
Lenkmoment
Lenkradmoment
Massenmoment
Massenträgheitsmoment
Mindestdrehmoment
Motordrehmoment
Motormoment
Nebenträgheitsmoment
Nennbremsmoment
Nenndrehmoment
Nennmoment
Pendelmoment
Phasenmoment
Potenzmoment
Quadrupolmoment
Rastmoment
Reibmoment
Reibungsmoment
Restmoment
Richtmoment
Rollmoment
Rückdrehmoment
Rückstellmoment
Sattelmoment
Schaltmoment
Schlupfmoment
Schnittmoment
Schwungmoment
Seitenmoment
Spitzendrehmoment
Stützmoment
Systemdrehmoment
Torsionsmoment
Trägheitsmoment
Widerstandsmoment
Winkelmoment
Zentrifugalmoment
Zerfallsmoment
Zugmoment
